src/appsettings.template.json

```diff
@@ -0,0 +1,40 @@
+{
+    "AppSettings": {
+        "BeginAllowedSchoolYear": 2022,
+        "EndAllowedSchoolYear": 2035,
+        "AuthenticationService": "http://dev-test-auth-service",
+        "DatabaseEngine": "postgresql",
+        "DeployDatabaseOnStartup": false
+    },
+    "ConnectionStrings": {
+        "DatabaseConnection": "host=${POSTGRES_HOST};port=${POSTGRES_PORT};username=${POSTGRES_USER};password=${POSTGRES_PASSWORD};database=edfi_datamanagementservice;"
+    },
+    "AllowedHosts": "*",
+    "RateLimit": {
+        "PermitLimit": 50,
+        "Window": 10,
+        "QueueLimit": 0
+    },
+    "Serilog": {
+        "Using": [ "Serilog.Sinks.File", "Serilog.Sinks.Console" ],
+        "MinimumLevel": {
+            "Default": "Information"
```

Comment:
This should probably be configurable by environment variable. Doesn't have to be in this ticket though. We can tackle this in DMS-194.